os-config.tpl.yml

```diff
@@ -105,7 +105,7 @@ rancher:
         io.rancher.os.detach: "false"
         io.rancher.os.reloadconfig: "true"
         io.rancher.os.scope: system
-        io.rancher.os.after: cloud-init-pre, wait-for-network
+        io.rancher.os.after: cloud-init-pre, wait-for-network, ntp
```

Comment:
Why do you depend on ntpd? The reason I ask is because if you need the system time to be set, there is no real guarantee it will be because we launch ntp in the background.